gitbook-docs/guides/environment-setup.md

```diff
@@ -1,33 +1,31 @@
 ---
-description: Environment setup for proof market cli
+description: Environment setup for the Proof Market CLI
 ---
 
 # Environment Setup
 
-In this guide, we set up the packages/libraries we require in order to interact with the proof market using command-line utilities.
+This guide describes how to set up packages/libraries, required to interact with the Proof Market, through the command line.
 
 ## Dependencies
 
-On \*nix systems, the following dependencies need to be present & can be installed using the following command.
+The required dependencies for \*nix systems can be installed via the following command:
 
 ```shell
 sudo apt install build-essential libssl-dev cmake clang-12 git autoconf libc-ares-dev libfmt-dev gnutls-dev liblz4-dev libprotobuf-dev libyaml-cpp-dev libhwloc-dev pkg-config xfslibs-dev systemtap-sdt-dev
 ```
 
 ### Boost
 
-Users need to install boost either manually or from their distros repository. Please ensure you are installing version 1.76. Follow the guide to install [version 1.76](https://www.boost.org/doc/libs/1\_76\_0/more/getting\_started/unix-variants.html) manually
+[Boost](https://www.boost.org) can be installed either manually or from the distributive's repository. Please ensure you are installing *version 1.76*. You can get it from the boost's [version history](https://www.boost.org/users/history/version_1_76_0.html).
```

Comment:
Please is used when users have to do something because of our failure.

```suggestion
[Boost](https://www.boost.org) can be installed either manually or from the distributive's repository.
Make sure you are installing *version 1.76*.
You can get it from the Boost's [version history](https://www.boost.org/users/history/version_1_76_0.html).
```